docs/tutorials/educators/intro.ipynb

```diff
@@ -1199,9 +1201,13 @@
     "\n",
     "The very first indication that quantum computers could be more powerful than classical computers was provided by David Deutsch in his 1985 paper\n",
     "\n",
-    "> David Deutsch,  \"[Quantum Theory, the Church-Turing Principle and the Universal Quantum Computer](https://people.eecs.berkeley.edu/~christos/classics/Deutsch_quantum_theory.pdf)\" *Proc. R. Soc. Lond.* A **400** 97–117. http://doi.org/10.1098/rspa.1985.0070\n",
+    "> David Deutsch,  \"[Quantum Theory, the Church-Turing Principle and the Universal Quantum Computer](https://people.eecs.berkeley.edu/~christos/classics/Deutsch_quantum_theory.pdf)\" *Proc. R. Soc. Lond. A* **400** 97–117 (1985). http://doi.org/10.1098/rspa.1985.0070\n",
```

Comment:
Does this need an {:external} or whatever?

Reply:
added let's see if it works